.circleci/config.yml

```diff
@@ -1,6 +1,40 @@
 version: 2.1
 orbs:
   elixir: membraneframework/elixir@1
+  win: circleci/windows@4.1
+
+jobs:
+  windows_test:
+    executor: 
+      name: win/default
+      size: medium
+      shell: cmd.exe
+
+    steps: 
+      - checkout
+
+      - restore_cache:
+          keys:
+            [choco-cache]
+          
+      - run:
+          name: "Build"
+          command: choco install -y erlang elixir
```

Comment:
For now this part is fine, but installing the Elixir every time seems costly (it takes more time than doing the actual tests). We should consider creating an image with the Elixir already installed - please add an issue to the https://github.com/membraneframework-labs/docker_membrane repo.